### PR TITLE
Community meeting

### PR DESCRIPTION
**Pull request by pwalczysko**
Adds an

- overview with the list of speakers.
- button on the main page encouraging to register
- ~~link on the Registration Open button is still pointing to the Events - TBD once we have the real link~~ - Edit: Done, the real link is included.

cc @jrswedlow @jburel @will-moore 

~~ToDo: Fix the "Register by 20th May" statement: This is off-the-cuff and maybe not what we want.~~ Edit: fixed, we want 21st May, this is implemented now.

built at https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ome-community-meeting-2024/ and 
https://snoopycrimecop.github.io/www.openmicroscopy.org/

### CODE REVIEW

**Comment by pwalczysko** [comment]
built at https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ome-community-meeting-2024/ and 
https://snoopycrimecop.github.io/www.openmicroscopy.org/

Note: The failing built is a chicke&egg problem - it remarks on non-existence of the pages which I want to create in this PR.

**Comment by pwalczysko** [comment]
I have added the eventbrite link to the ome-community-meeting-program-2024.html and implemented @will-moore 's comments.

File: _includes/ome2024_program_agenda.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    <p>
+        This schedule is subject to change and will be updated periodically.<br/>
+        The schedule will include pre-recorded videos organised by topic followed by group discussions.
+    </p>
+</div>
+
+<div class="row column text-center">
+<h4>Schedule</h4>
+<p>
+    The content will repeated over two sessions to accomodate our
```
[the remark this answers is not in the document]
Comment (pwalczysko) [reply _includes/ome2024_program_agenda.html]: Yes, thanks, I removed this file. It was copied from OME 2021 - and, ultimately, after reduction, I do not use it. Will return it once we have programme.

fixed in https://github.com/ome/www.openmicroscopy.org/pull/708/commits/990280ddca5fcacadff59543c9d8e4d171cd5cf0

File: _includes/ome2024_program_agenda.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    <p class="subheader">{{ event.description }}</p>
+</div>
+<hr>
+<hr class="invisible">
+
+<div class="row column text-center">
+    <h5>Please Note</h5>
+    <p>
+        This schedule is subject to change and will be updated periodically.<br/>
+        The schedule will include pre-recorded videos organised by topic followed by group discussions.
```
[the remark this answers is not in the document]
Comment (pwalczysko) [reply _includes/ome2024_program_agenda.html]: No, and thanks, I removed this file. It was copied from OME 2021 - and, ultimately, after reduction, I do not use it. Will return it once we have programme.

fixed in https://github.com/ome/www.openmicroscopy.org/pull/708/commits/990280ddca5fcacadff59543c9d8e4d171cd5cf0

**Comment by pwalczysko** [comment]
@jburel I removed the confusing pages which were copied from OME 2021 and included "online format" mentions. Note that nothing changed on https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ome-community-meeting-2024/ as expected.

File: events/ome-community-meeting-2024/index.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                    <p>
+                    {% for speaker in speakers %}
+                    <a target="_blank" href="{{ speaker.sameAs }}">
+                    {{ speaker.givenName }} {{ speaker.familyName }}
+                    </a>
+                    <span class="institution">{{ speaker.affiliation.name }}</span></br>
+                    {% endfor %}
+                    </p>
+
+                    <p class="card-caption margin-top">We hope that this structure can accommodate the schedules of different members of the community. We’ll continue to post updates here, on <a href="https://twitter.com/openmicroscopy" target="_blank">Twitter (@openmicroscopy)</a>, and the <a href="https://forum.image.sc/" target="_blank">image.sc forum</a> and we look forward to meeting you all soon.</p>
```
Comment (jburel) [inline events/ome-community-meeting-2024/index.html:63]: Maybe use X instead of Twitter

Comment (pwalczysko) [reply events/ome-community-meeting-2024/index.html]: fixed in https://github.com/ome/www.openmicroscopy.org/pull/708/commits/271207c153bbae585a4a0a62bbbacca6a0b140a3

**Comment by pwalczysko** [comment]
-- exclude

**Comment by pwalczysko** [comment]
Build passes

**Comment by pwalczysko** [comment]
--include

**Comment by pwalczysko** [comment]
Merging as okayed by June in slack.

**Pull request merged**